devDocs/buildSystemNotes.md

```diff
@@ -0,0 +1,23 @@
+A Python virtual environment is now used transparently by the NVDA build system, and all Python dependencies are installed into this environment using `pip`.

```

Comment:
Rather than "now" I'd make this a statement about the current behavior, it is more simple to read the concrete current truth rather than interpreting a series of changes.

```suggestion
A Python virtual environment is used transparently by the NVDA build system, and all Python dependencies are installed into this environment using `pip`.

```